simulcast.html

```diff
@@ -0,0 +1,410 @@
+      event</a></dfn> and <dfn><a href=
+      "http://dev.w3.org/html5/spec/webappapis.html#networking-task-source">networking
+      task source</a></dfn> are defined in [[!HTML51]].</p>
+      <p>The terms <dfn>event</dfn>, <dfn><a href=
+      "http://dev.w3.org/html5/spec/webappapis.html#event-handlers">event
+      handlers</a></dfn> and <dfn><a href=
+      "http://dev.w3.org/html5/spec/webappapis.html#event-handler-event-type">event
+      handler event types</a></dfn> are defined in [[!HTML51]].</p>
+     <p>When referring to exceptions, the terms <dfn><a
+      href="https://www.w3.org/TR/WebIDL-1/#dfn-throw">throw</a></dfn> and
+      <dfn data-dfn-for="exception"><a href=
+      "https://www.w3.org/TR/WebIDL-1/#dfn-create-exception">create</a></dfn> are
+      defined in [[!WEBIDL-1]].</p>
+      <p>The terms <dfn data-lt="fulfill|fulfillment">fulfilled</dfn>, <dfn
+      data-lt="reject|rejection|rejecting">rejected</dfn>,
+      <dfn data-lt="resolve|resolves">resolved</dfn>, <dfn>pending</dfn> and
+      <dfn>settled</dfn> used in the context of Promises are defined in
+      [[!ECMASCRIPT-6.0]].</p>
+      <p>The terms <dfn>MediaStream</dfn>, <dfn>MediaStreamTrack</dfn>, and
+      <dfn>MediaStreamConstraints</dfn> are defined in [[!GETUSERMEDIA]].</p>
+      <p>This specification references objects, internal slots and dictionaries defined in
+      [[!WEBRTC]], including the <dfn><code>RTCRtpEncodingParameters</code></dfn> dictionary
+      (defined in Section 5.2.6), the <dfn><code>RTCRtpTransceiver</code></dfn> object,
+      its <dfn>[[\Sender]]</dfn> and <dfn>[[\Receiver]]</dfn> internal slots (defined
+      in Section 5.4), the <dfn><code>RTCRtpSender</code></dfn> object, its
+      <dfn>[[\SendEncodings]]</dfn> and <dfn>[[\LastReturnedParameters]]</dfn> internal slots (defined in Section 5.2),
+      and the <dfn><code>RTCRtpReceiver</code></dfn> object (defined in Section 5.3).</p>
+  </section>
+  <section id="rtcrtpcodingparameters">
+    <h3><dfn>RTCRtpCodingParameters</dfn> Dictionary</h3>
+    <div>
+      <pre class="idl">partial dictionary RTCRtpCodingParameters {
+           DOMString           rid;
+};</pre>
+      <section>
+        <h2>Dictionary <code><a>RTCRtpCodingParameters</a></code>
+        Members</h2>
+        <dl data-link-for="RTCRtpCodingParameters" data-dfn-for=
+        "RTCRtpCodingParameters" class="dictionary-members">
+          <dt><dfn data-idl><code>rid</code></dfn> of type <span class=
+          "idlMemberType"><a>DOMString</a></span></dt>
+          <dd>
+            <p>If set, this RTP encoding will be sent with the RID header
+            extension as defined by <span data-jsep=
+            "initial-offers">[[!JSEP]]</span>. The RID is not modifiable via
+            <code>setParameters</code>. It can only be set or modified in
+            <code>addTransceiver</code> on the sending side.
+            <a>Read-only parameter</a>.</p>
```

Comment:
Possible future cleanup: There's a mix between what a dictionary member represents ("what's rid?") and how a specific method/algorithm uses it. I think stuff like what setParameters and addTransceivers does belongs to the steps executed when calling setParameters or addTransceiver, not to the dictionary description.

Reply:
Yes.